client/src/styles/login-form.scss

```diff
@@ -17,81 +19,85 @@ html {
     padding: var(--spacing-form-desktop);
     width: auto;
   }
-}
 
-.login-form__title {
-  margin-top: 0;
-}
+  &__title {
```

Comment:
Not super keen on increasing the specificity of these rules to be honest. If it's a convention followed elsewhere in core I guess it makes sense to follow it, but to me the closer we can get to writing standard CSS, the easier it'll make deprecating SCSS in the future when it's no longer required.

Reply:
Ok, _some_ of this specificity was required to override core styles, but perhaps we could call those out individually rather than wrapping the whole thing in another nesting level...?

Reply:
Oh wait, I misunderstood - are you suggesting we shouldn't break up the block and element part of the class name like this? You're right that the convention in core is to do this rather than not to

Reply:
I pushed an update to remove the specificity and class name concatenation for all but the `input.text` style, which needs it